### DISCUSSION TITLE
Problem finding `references.bib` in a subdirectory

### DISCUSSION THREAD

**Discussion by JoFrhwld** [quarto-dev/quarto-cli#2957] [Q&A]
I'm trying to render a quarto website that has a subdirectory with a qmd and a bib. But I've been getting an error from pandoc that reads:

```
ERROR: Failed to generate bibliography, rendering failed with code 1
pandoc: <PATH1>/<PATH2>references.bib: openBinaryFile: does not exist (No such file or directory)
```
Where `<PATH1>` is the full path to the website's top directory, and `<PATH2>` is the full path to the bib file, concatenated onto the first path.

UPDATE!

I've figured out that the problem is somehow related to `google-scholar: true` in the main site's `_quarto.yml`. This is re-creating the issue for me:

```bash
git clone https://github.com/JoFrhwld/big_ref.git
cd big_ref
quarto render
```

If I edit the `_quarto.yml` file inside the project to comment out `google-scholar: true` the site renders fine (both my actual site and this minimal example)

-----
Previous Version

I tried recreating a minimal example [here](https://github.com/JoFrhwld/big_ref), but that website renders without an issue. Unfortunately, getting my specific example working is a little complicated, but here's how to do it:

```bash
git clone https://github.com/LingMethodsHub/LingMethodsHub.github.io.git
cd LingMethodsHub.github.io
git checkout drafting-tidynorm
git submodule update --init content/R/tidy-norm
cd content/R/tidy-norm
```

Then (because I'm still working out how to use renv) you'll need to launch R, and run

```r
renv::init()
## Choose to set up from the lockfile
renv::install("concaveman")
## For some reason I could not get this package into the lockfile
quit()
```

Then

```bash
cd ../../../
quarto render
```

Then, you should get the error. 

Just for information about my current quarto and pandoc versions:

```bash
❯ quarto --version
1.2.210
❯ pandoc --version
pandoc 2.19.2
Compiled with pandoc-types 1.22.2.1, texmath 0.12.5.2, skylighting 0.13,
citeproc 0.8.0.1, ipynb 0.2, hslua 2.2.1
Scripting engine: Lua 5.4
```

**Comment by JoFrhwld** [comment]
Some additional things I've tried, that didn't change the error:

- I changed bibliography value to `bibliography: "./references.bib"` in the content page header
- I removed `csl: assets/lingmethodshub.csl` in the main site `_quarto.yml`
- I tried setting `site-path: "."` in the content folder's `_quarto.yml`

In all cases, I still got the same `pandoc: <PATH1>/<PATH2>references.bib: openBinaryFile: does not exist (No such file or directory)` error

**Reply by dragonstyle** [reply]
I was able to follow your clear instructions above to get setup and running (I did run into an error in the last code cell in authors/index.qmd but I think that is unrelated so I just removed that code cell for testing purposes).  Thank you for sharing that so clearly!

Unfortunately, rendering didn't cause an error for me. I can view the `tidy-norm/index.qmd` page rendered without issue (and with the references properly compiled into a bibliography at the bottom of the page). Am I looking in the right place for the error?

**Reply by JoFrhwld** [reply]
Hi! I'm updating the original post because I worked out that the error is being caused by `google-scholar: true` in the main site's `_quarto.yml` and have a reproducible example.

**Reply by dragonstyle** [reply]
Ah this is a great piece of info - thank you!

I made a change in just this area which should be included in any build past v1.2.215  - do you mind trying the latest pre-release and see if that resolved the issue? https://quarto.org/docs/download/prerelease.html

**Reply by dragonstyle** [reply]
(I just tried cloning your simple repo and it was fixed for me using the latest pre-release, so I'm hoping my other fix resolved it!)

**Reply by JoFrhwld** [reply]
I’ll try this out with the bigger site later today!

**Reply by dragonstyle** [reply]
Great, thank you!

**Reply by JoFrhwld** [reply]
Yup! Quarto v1.2.233 renders without the pandoc error with `google-scholar: true` enabled!
